.circleci/config.yml

```diff
@@ -759,7 +759,7 @@ jobs:
       - run: *desktop-install-app-deps
       - run:
           name: Build Desktop Mac
-          no_output_timeout: 30m
+          no_output_timeout: 45m
```

Comment:
🤯

Reply:
I've observed notarization can take as long as 10-15 minutes turnaround time. Honestly a ~30m no output timeout should be fine, but just in case ... 😬

Reply:
Better safe than sorry, yeah 👍